pkg/utils/maps/maps.go

```diff
@@ -197,3 +197,24 @@ func SerializeMap(m map[string]string) string {
 func DeSerializeCache(s string) []string {
 	return strings.Split(s, ",")
 }
+
+// GetNestedField returns the nested field of a map.
+// Example: GetNestedField(map[string]interface{}{"a": map[string]interface{}{"b": "c"}}, "a.b") returns "c".
+func GetNestedField(m map[string]interface{}, path string) (interface{}, error) {
```

Comment:
In modern go, I think `any` type is identical to `interface{}` and more idiomatic. Not entirely sure about this